src/functions/createModal.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @author welvin21
+ * This function creates a modal
+ * based on a unique key which determines the modal message
+ * @param  {string} modalKey modal key which is unique to a particular modal message
+ * @return {object}          an object describing the details of the modal
+ */
+export const createModal = modalKey => {
```

Comment:
probably should rename this to `createModalMessage` instead of `createModal` which can be confused as a function that creates a react component

Reply:
alrightt, will make the changes in a moment